README.md

```diff
@@ -74,13 +86,30 @@ device_manager { 'cisco.example.com':
 
 (Using the `device_manager` module will also automatically classify the proxy Puppet agent with the `cisco_ios` class.)
 
-The following additional `credentials` keys are optional:
+To save the running config, it is possible to use the `cisco_ios::config_save` task. Before running this task, install the module on your machine, along with [Puppet Bolt](https://puppet.com/docs/bolt/latest/bolt_installing.html). When complete, execute the following command:
```

Comment:
This section would probably be best under a new/different section, it feels weird talking about setting up the device configuration to then start talking about a task without even testing the setup you've just done for `puppet device`

Reply:
yeah, good spot.